inference-engine/thirdparty/clDNN/src/gpu/configuration.cpp

```diff
@@ -24,6 +29,14 @@ configuration::configuration()
       queues_num(0),
       tuning_cache_path("cache.json"),
       kernels_cache_path(""),
-      n_threads(std::max(static_cast<uint16_t>(std::thread::hardware_concurrency()), static_cast<uint16_t>(1))) {}
+      n_threads(std::max(static_cast<uint16_t>(std::thread::hardware_concurrency()), static_cast<uint16_t>(1))),
+      core_type(-1 /*task_arena::automatic*/) {
+#if (CLDNN_THREADING == CLDNN_THREADING_TBB)
+      if ((cldnn::custom::info::core_types().size() > 1 /*Hybrid CPUs*/)) {
+          n_threads = std::min(static_cast<uint16_t>(InferenceEngine::getNumberOfCPUCores(true)), n_threads);
```

Comment:
Maybe can we use `tbb::this_task_area::max_concurrency()` and avoid the linking with inference_engine?

Reply:
I use getNumberOfCPUCores to get the number of only big cores instead of getting max concurrency. but I got an idea from your comments and update patch to remove liking inference_engine. thank you for your comment.